src/lib/scheduler/operator_task.cpp

```diff
@@ -91,5 +93,23 @@ void OperatorTask::_on_execute() {
 
     context->rollback();
   }
+
+  // Get rid of temporary tables that are not needed anymore
+  if (_cleanup_temporaries) {
```

Comment:
Doc that the nature of the successor-clears-predecessor mechanism makes sure that the root operator's output is never cleared and that this is (obviously) intended

Reply:
Yepp